src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public FruitTransaction() {

```

Comment:
The default constructor is not necessary if it's not going to be used explicitly. If there are no other constructors, Java provides a default one automatically.